invoices/invoiceregistry.go

```diff
@@ -402,7 +409,9 @@ func (i *InvoiceRegistry) deliverSingleBacklogEvents(
 
 	// It is possible that the invoice does not exist yet, but the client is
 	// already watching it in anticipation.
-	if err == channeldb.ErrInvoiceNotFound {
+	if err == channeldb.ErrInvoiceNotFound ||
+		err == channeldb.ErrNoInvoicesCreated {
```

Comment:
nice catch, is this to handle the case of when your node is brand new?

Reply:
Yes. But it actually surfaced in a unit test. I am not sure if a separate error for no bucket is justified though, but didn't want to go into that in this pr.